keen/keen-dataviz.js
# Escape characters where appropriate

## What does this PR do?
It adds escaping of characters in the appropriate places before presenting the values in the visualizations and visualization wrapper.

## How can I test this?
1. Ensure all tests pass.
2. Try adding `<script>alert('hi');</script>` in data values and ensure they are properly escaped for all chart types.
